src/Files.App/Converters/BoolToGlyph.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Files.App.Converters
+{
+	public class RecentFilesBoolToGlyph
```

Comment:
Please rename the file as well

Reply:
Honestly, I'd make it a generic glyph converter where the converter parameter is the glyph:
`<glyphIfTrue>|<glyphIfFalse>`

Reply:
Ok, so I'll change this and I'll add a comment in the xaml to explian the two glyphs

Reply:
> Honestly, I'd make it a generic glyph converter where the converter parameter is the glyph:

That's what I originally intended, thought we had a converter for this already.

Reply:
![1](https://user-images.githubusercontent.com/102259289/193297064-98060b8b-a9ab-4d3c-9981-028ec568a2b1.png)
So, should I use the built-in converter (like in the picture) or should I add a brand new converter?

Reply:
The built in one looks good, that's what I had in mind.